lib/id3c/api/datastore.py

```diff
@@ -259,35 +259,96 @@ def fetch_identifier_set(session: DatabaseSession, name: str) -> Any:
 def make_identifier_set(session: DatabaseSession, name: str, **fields) -> bool:
     """
     Create a new identifier set *name* in the backing database using *session*
-    if it doesn't already exist.
+    if it doesn't already exist, or update if it does exist.
 
-    If *description* is provided as a keyword argument, its value is
-    set/updated in the database.
+    If *use* and/or *description* are provided as keyword arguments, their values
+    are set in the database. Becuase *use* is a required field in the target table,
+    if it is not provided as a keyword argument the query will attempt to retrieve
+    its value from an existing record.
```

Comment:
That would have been my first inclination, to create separate endpoints for creates and updates, but I was following the pattern already established for the optional `description` argument here:
https://github.com/seattleflu/id3c/blob/ca2d7f127b8e86818a469ba18b74aeef5e05844d/lib/id3c/api/datastore.py#L264-L268

There was some previous discussion on this PR about including `use` as a keyword argument, but it isn't showing up any more because I marked it as resolved (I'm inclined to _not_ do that going forward)